src/main/webapp/resources/js/pages/analysis/components/AnalysisProvenance.jsx

```diff
@@ -1,12 +1,206 @@
-import React from "react";
-import { Typography } from "antd";
+/**
+ * @File component renders the analysis provenance for the
+ * file the user has selected to view
+ */
 
-const { Title } = Typography;
+import React, { useEffect, useContext, useState } from "react";
+import { Collapse, Descriptions, Layout } from "antd";
+import { SPACE_MD } from "../../../styles/spacing";
+import {
+  grey1,
+  grey3,
+  blue2,
+  blue3,
+  blue4,
+  blue5,
+  blue6,
+  blue7,
+  blue8,
+  blue9,
+  blue10
+} from "../../../styles/colors";
+import { getAnalysisProvenanceByFile } from "../../../apis/analysis/analysis";
+import { TabPaneContent } from "../../../components/tabs/TabPaneContent";
+import { AnalysisContext } from "../../../contexts/AnalysisContext";
+import { AnalysisOutputsContext } from "../../../contexts/AnalysisOutputsContext";
+
+const { Panel } = Collapse;
 
 export default function AnalysisProvenance() {
+  const { analysisContext } = useContext(AnalysisContext);
+  const { analysisOutputsContext, getAnalysisOutputs } = useContext(
+    AnalysisOutputsContext
+  );
+  const [provenance, setProvenance] = useState(null);
+  const [toolInfo, setToolInfo] = useState([]);
+  const [currFileName, setCurrFileName] = useState(null);
+
+  const toolPanelMargin = 5;
+  const levelColors = [
```

Comment:
Add a comment as to why you are using this structure.

Reply:
Added in 7165a33809647dafedbd205e730437decb47dde8